sw/device/tock/dif/build.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+extern crate bindgen;
+
+use bindgen::builder;
```

Comment:
I would avoid a line like this. `builder()` below looks pretty confusing, since it's not clear where it came from, and I think I would have written `bindgen::builder()`. In general, it's best to only import stuff you wind up using a lot.

Reply:
While I understand that this is a small file and this line doesn't cause a lot of ambiguity, I agree with @mcy. From https://doc.rust-lang.org/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html:
> Bringing the function’s parent module into scope with use so we have to specify the parent module when calling the function makes it clear that the function isn’t locally defined while still minimizing repetition of the full path.

Reply:
Done.